src/routes/userRouter.ts

```diff
@@ -0,0 +1,9 @@
+import { Router } from "express";
+import userControllers from "../modules/user/controller/userControllers";
+import { validateUserId, validateUserStatus, checkUserExists } from "../modules/user/validation/userValidations";
+
+const router: Router = Router()
+
+router.put("/update-user-status/:id", validateUserId, validateUserStatus, checkUserExists, userControllers.updateUserStatus);
```

Comment:
This `checkUserExists` function was supposed to be into middleware because it is working as middleware. Also check if we don't have a function which is working the same job.